onnx_coreml/_operators_nd.py

```diff
@@ -122,6 +146,83 @@ def _convert_gather(builder, node, graph, err):
         axis=axis
     )
 
+def _convert_gemm(builder, node, graph, err):
+    '''
+    convert to CoreML Tranpose (Optional) and Inner Product Layer:
+    https://github.com/apple/coremltools/blob/655b3be5cc0d42c3c4fa49f0f0e4a93a26b3e492/mlmodel/format/NeuralNetwork.proto#L4180
+    '''
+    # Read attributes
+    alpha = node.attrs.get('alpha', 1.0)
+    beta = node.attrs.get('beta', 1.0)
+    transA = node.attrs.get('transA', False)
+    transB = node.attrs.get('transB', False)
+
+    A = node.inputs[0]
+    if alpha != 1.0:
+        builder.add_load_constant_nd(
+            name=node.name + '_load_alpha',
+            output_name='alpha_for_'+A,
+            constant_value=alpha,
+            shape=[1]
+        )
+        builder.add_multiply_broadcastable(
+            name=node.name + '_alphaA',
+            input_names=[A, 'alpha_for_'+A],
+            output_name=A+'_alphaA'
+        )
+        A = A + '_alphaA'
+
+    B = node.inputs[1]
+    C = node.inputs[2]
+    if B in node.input_tensors and C in node.input_tensors:
+        B = node.input_tensors[B]
+        C = node.input_tensors[C]
+
+        if transB:
+            B = B.transpose()
+        
+        C = C.flatten()
+        builder.add_batched_mat_mul(
+            name=node.name,
+            input_names=[A],
+            output_name=node.outputs[0],
+            transpose_a=transA,
+            weight_matrix_rows=B.shape[0],
+            weight_matrix_columns=B.shape[1],
+            W=B,
+            bias=C
+        )
+    else:
+        ## TODO: Test coverage when B and C are non-constant
```

Comment:
Do we need to handle case when W and B are non-constant?
The only reason of this not being true is not doing constant propagation correctly.